app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -129,4 +148,14 @@ public void setCategories(List<Category> categories) {
     public void setContestDescription(String description) {
         presenter.setContestDescription(description);
     }
+
+    @Override
+    public void onNextDisabled() {
```

Comment:
lets change these to setNextEnabled/Disabled. We could also reduce to one function and pass in a boolean.